app/models/consent_form.rb

```diff
@@ -196,7 +194,7 @@ class ConsentForm < ApplicationRecord
   end
 
   def address_postcode=(str)
-    super UKPostcode.parse(str.to_s).to_s
+    super str.nil? ? nil : UKPostcode.parse(str.to_s).to_s
```

Comment:
Would this have the same effect? Would `str` ever be false?

```suggestion
    super str || UKPostcode.parse(str.to_s).to_s
```

Reply:
Hmm doesn't this do the opposite of what we need? E.g. it wouldn't parse and reformat `str` if it's truthy.